### PR TITLE
VACMS-6536: Make inline national taxonomy service descriptions visible on edit

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Relates to #6536

## Testing done
Visual

## QA steps

Vet Center service
- [ ] As an admin (the experience is the same for any role), go to [/node/3690/edit](https://pr6960-urxnmybr9vlth6fbfpja57qqsr5rnzfv.ci.cms.va.gov/node/3690/edit) and click the edit button for an existing service.
- [x] Visually verify the service term fields load when the service loads
- [x] Visually verify help text at top of fieldset is appended with `(opens in a new tab)`
- [ ] Visually verify that `(opens in a new tab)` is part of the hyperlink

VAMC system service
- [ ] Go to [/node/28505/edit](https://pr6960-urxnmybr9vlth6fbfpja57qqsr5rnzfv.ci.cms.va.gov/node/28505/edit) 
  - [x] visually verify help text at top of page is appended with `(opens in a new tab)`
  - [ ] Visually verify that `(opens in a new tab)` is part of the hyperlink
  - [x] See that the General service description is visible
  - [ ] Go to [/node/add/regional_health_care_service_des](https://pr6960-urxnmybr9vlth6fbfpja57qqsr5rnzfv.ci.cms.va.gov/) and see that it's consistent on the create form too.

### CODE REVIEW

**Review by netboss — DISMISSED** [review]
@ethanteague This looks good!

File: config/sync/field.field.node.vet_center.field_health_services.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ field_name: field_health_services
 entity_type: node
 bundle: vet_center
 label: Services
-description: 'Add services that Veterans can receive at your Vet Center. Some content won’t be editable because it comes from other sources. For full guidance, see <a target="_blank" href="https://prod.cms.va.gov/help/vet-centers/how-to-edit-a-vet-center-service">How to add or edit a Vet Center service.</a>'
+description: 'Add services that Veterans can receive at your Vet Center. Some content won’t be editable because it comes from other sources. For full guidance, see <a target="_blank" href="https://prod.cms.va.gov/help/vet-centers/how-to-edit-a-vet-center-service">How to add or edit a Vet Center service.</a>  (opens in a new tab)'
```
Comment (suzanne-gray) [inline config/sync/field.field.node.vet_center.field_health_services.yml:21]: Please make `(opens in a new tab)` part of the hyperlink. The period at the end of the sentence should not be linked.

**Comment by suzanne-gray** [comment]
In the `Vet center service description` field label, Center should be capitalized

**Comment by suzanne-gray** [comment]
One thing that's a little funky here - it seems to me that when editing, the service name should not have a dropdown (because you wouldn't be trying to change the name to something else). Also the "required" red asterisk doesn't really make sense in this context. Can we have it just display the field without the dropdown and asterisk?  @kevwalsh thoughts? 

Screen Shot 2021-11-12 at 10 46 10 AM

**Comment by kevwalsh** [comment]
i agree 100% with @suzanne-gray  that once the service has been created, the service dropdown should be disabled (or even better, rendered as an h3 or something). But i think it should go into a separate issue because there may be some tricky tech considerations for that, and i wouldn't want it to hold up this improvement.

**Comment by ethanteague** [comment]
> i agree 100% with @suzanne-gray that once the service has been created, the service dropdown should be disabled (or even better, rendered as an h3 or something). But i think it should go into a separate issue because there may be some tricky tech considerations for that, and i wouldn't want it to hold up this improvement.

Everything else updated :+1:

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**